org.gnome.Eolie.json

```diff
@@ -0,0 +1,314 @@
+{
+  "app-id": "org.gnome.Eolie",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "command": "eolie",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=host",
```

Comment:
Eolie needs to access ~/.mozilla and ~/.config/chromium ... Do I need to reenable this option?